package.json

```diff
@@ -1,69 +1,61 @@
-    "@testing-library/user-event": "^14.6.1",
-    "@types/lodash": "^4.17.15",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@wordpress/browserslist-config": "^6.19.0",
-    "eslint": "^8.57.0",
-    "html-entities": "^2.5.2",
-    "lint-staged": "^15.4.3",
-    "mini-css-extract-plugin": "^2.9.2",
-    "newspack-components": "^3.1.0",
-    "newspack-scripts": "^5.5.2",
-    "postcss-focus-within": "^9.0.1",
-    "postcss-scss": "^4.0.9",
-    "webpack-fix-style-only-entries": "^0.6.1"
-  },
-  "dependencies": {
-    "move-file-loader": "^1.0.0",
-    "npm-run-all": "^4.1.5"
-  }
-}
+	"name": "@automattic/newspack-block-theme",
+	"version": "1.14.0",
+	"description": "A forward-looking news theme designed and developed to be highly customizable with the WordPress block editor. Brought to you by Newspack: https://newspack.com",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/newspack-block-theme.git"
+	},
+	"author": "Newspack",
+	"license": "GPL-3.0",
+	"bugs": {
+		"url": "https://github.com/Automattic/newspack-block-theme/issues"
+	},
+	"homepage": "https://github.com/Automattic/newspack-block-theme#readme",
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
+	],
+	"lint-staged": {
+		"*.scss": "npm run lint:scss:staged",
+		"*.(js|jsx)": "npm run lint:js:staged",
+		"*.php": "npm run lint:php:staged"
+	},
+	"scripts": {
+		"cm": "newspack-scripts commit",
+		"semantic-release": "semantic-release",
+		"start": "npm ci --legacy-peer-deps && npm run watch",
+		"clean": "rm -rf dist ; rm -f style.css",
+		"build": "npm run clean && newspack-scripts wp-scripts build",
+		"watch": "npm run clean && newspack-scripts wp-scripts start",
+		"lint": "npm run lint:scss && npm run lint:js",
+		"lint:js": "newspack-scripts wp-scripts lint-js '**/src/**/*.{js,jsx}'",
+		"lint:js:staged": "newspack-scripts wp-scripts lint-js --ext .js,.jsx",
+		"fix:js": "newspack-scripts wp-scripts lint-js --fix '**/src/**/*.{js,jsx}'",
+		"format:js": "newspack-scripts wp-scripts format '**/src/**/*.{js,jsx}'",
+		"lint:php": "./vendor/bin/phpcs",
+		"lint:php:staged": "./vendor/bin/phpcs --filter=GitStaged",
+		"fix:php": "./vendor/bin/phpcbf",
+		"lint:scss": "newspack-scripts wp-scripts lint-style '**/src/**/*.scss' --customSyntax postcss-scss",
+		"lint:scss:staged": "newspack-scripts wp-scripts lint-style --customSyntax postcss-scss",
+		"format:scss": "newspack-scripts wp-scripts lint-style '**/src/**/*.scss' --customSyntax postcss-scss --fix",
+		"release:archive": "rm -rf release && mkdir -p release && rsync -r . ./release/newspack-block-theme --exclude-from='./.distignore' && cd release && zip -r newspack-block-theme.zip newspack-block-theme",
+		"release": "npm run build && npm run semantic-release"
+	},
+	"devDependencies": {
+		"@rushstack/eslint-patch": "^1.10.5",
+		"@testing-library/dom": "^10.4.0",
+		"@testing-library/user-event": "^14.6.1",
+		"@types/lodash": "^4.17.15",
```

Comment:
`lodash` is not used in this project, this can be removed.

Reply:
Yep, removed in 8240d98